bin/node-template/runtime/build.rs

```diff
@@ -6,5 +6,6 @@ fn main() {
 		.with_wasm_builder_from_crates("2.0.0")
 		.export_heap_base()
 		.import_memory()
+		.append_to_rust_flags("--cfg feature=\\\"runtime-wasm\\\"")
```

Comment:
Why?

Reply:
from the errors I concluded that the feature `runtime-wasm` isn't applied automatically by the build script. Is that wrong?

Reply:
When the feature is present in the cargo.toml, the wasm-builder should activate it automatically.